Tools/TestWebKitAPI/Tests/WebKitCocoa/SafeBrowsing.mm

```diff
@@ -24,6 +24,7 @@
  */
 
 #import "config.h"
+#include "gtest/gtest.h"
```

Comment:
In .mm files we import.  This is also out of place.  This shouldn't be necessary.